src/ui/lib/components/header.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+	import logo from '@images/zarf-logo.png';
+</script>
+
+<header class="header">
```

Comment:
This seems like it should use https://github.com/material-components/material-components-web/tree/master/packages/mdc-top-app-bar maybe?

Reply:
ill play around w/ it